pkg/gui/controllers/switch_to_sub_commits_controller.go

```diff
@@ -42,6 +44,11 @@ func (self *SwitchToSubCommitsController) GetKeybindings(opts types.KeybindingsO
 			Key:         opts.GetKey(opts.Config.Universal.GoInto),
 			Description: self.c.Tr.ViewCommits,
 		},
+		{
+			Handler:     self.viewLeftRightCommits,
```

Comment:
This controller is attached to a few contexts:
```
		gui.State.Contexts.Branches,
		gui.State.Contexts.RemoteBranches,
		gui.State.Contexts.Tags,
		gui.State.Contexts.ReflogCommits,
```
But we only want this keybinding for branches right?

Reply:
Right. In the implementation we check whether the model is a `models.Branch` and only do the left-right thing in that case, falling back to the normal log if not. We also fall back to normal log if the branch has no upstream, so we need those if statements anyway.